index.html

```diff
@@ -906,66 +944,67 @@ <h2>Attributes</h2>
               include framing overhead incurred by the protocol, or buffering done
               by the operating system or network hardware. On getting, it
               <em class="rfc2119" title="MUST">MUST</em> return the value of the
-              <code><a>RTCQuicStream</a></code>'s <a>[[\WriteBufferedAmount]]</a> internal slot.
-              If the <code><a>RTCQuicStream</a></code> is in the <code>closed</code>
-              state, this attribute's value will only increase with each call to the
-              <code>write</code> method (the attribute does not reset to zero once the
-              <code><a>RTCQuicStream</a></code> closes).</p>
+              RTCQuicWritableStream's <a>[[\WriteBufferedAmount]]</a> internal slot.
+            </dd>
+            <dt><code>writingAborted</code> of type <span class=idlAttriType"><a>RTCQuicStreamAbortInfo</a>
+            readonly</dt>
+            <dd>
+              <p>The <dfn id="dom-rtcquicwritablestream-writingAborted"><code>writingAborted</code></dfn>
+              attribute represents a promise that <a>resolves</a> when the
+              STOP_SENDING frame is received from the <code><a>RTCQuicReadableStream</a></code>.
+              When the <var>stream</var> receives a STOP_SENDING frame from its corresponding
+              <code><a>RTCQuicReadableStream</a></code>, the <a>user agent</a>

```

Comment:
I'm unclear on terminology here. If the stream is a bidirectional stream, it has both the writable and readable stream as mixins - which means that the ReadableStream isn't a corresponding stream, it's the same stream. And if it's a one-way Stream, it has no ReadableStream mixin, so this can't happen.

Is this a property of bidirectional streams only?

Reply:
STOP_SENDING is sent from a readable stream to a writable stream to signal a "hard shutdown."
https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#rfc.section.7.15

I'll try to clarify for both cases, and address a potential bug you called out. 
**bidirectional stream**
This will have both readable and writable stream as mixins (let's call this side local). When the remote side's readable stream mixin (as part of the remote side's RTCQuicBidirectionalStream) sends a STOP_SENDING frame, it will be received by 
the local side's writablestream mixin. 
**one-way stream**
The local side is a writable stream, the remote side a readable stream. The writable stream can receive a STOP_SENDING frame from the corresponding readable stream on the remote side.
**bug**
The RTCQuicBidirectionalStream is added to the [[QuicTransportStreams]] slot with createStream/onnewbidirectionalstreamwithdata. I think what really needs to be specified is that both it's underlying mixins (readable and writable) streams are added to the slot [[QuicTransportStreams]]. This way they can be individually removed with abortWriting/writingAborted and abortReading/readingAborted.

Reply:
Or just have [[QuicTransportSendStreams]] and [[QuicTransportReceiveStreams]], with Bidirectional streams getting added to both.

Reply:
I like Steve's idea here. I went ahead and made the change.